core/src/sumeragi/main_loop.rs

```diff
@@ -33,6 +33,8 @@ pub struct Sumeragi {
     pub kura: Arc<Kura>,
     /// [`iroha_p2p::Network`] actor address
     pub network: IrohaNetwork,
+    /// Receiver channel, for control flow messages.
+    pub control_message_receiver: mpsc::Receiver<MessagePacket>,
```

Comment:
```suggestion
    pub control_message_receiver: mpsc::Receiver<ViewChangeSuggested>,
```

we want it to be a separate type not a variant of `Message`